### PR TITLE
Support TorchServe

### PR DESCRIPTION
**Pull request by hudeven**
Add handler and instruction for torchserve

### CODE REVIEW

File: torchrecipes/paved_path/serve/handler.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    #     return data
+
+    def _load_torchscript_model(self, model_pt_path):
+        """Loads the PyTorch model and returns the NN model object.
+        Args:
+            model_pt_path (str): denotes the path of the model file.
+        Returns:
+            (NN Model Object) : Loads the model object.
+        """
+        # TODO: remove this method if https://github.com/pytorch/text/issues/1793 gets resolved
```
Comment (dracifer) [inline torchrecipes/paved_path/serve/handler.py:100]: "this method" refers to the separate call of `torch.jit.load()` and `model.to` and replace it with `torch.jit.load(model_pt_path, map_location=self.device)`?

File: torchrecipes/paved_path/serve/handler.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
+"""
+Module for text classification with scriptable tokenizer
```
Comment (dracifer) [inline torchrecipes/paved_path/serve/handler.py:2]: this is the new code and other files are from PR#36?

File: torchrecipes/paved_path/serve/handler.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        line = data[0]
+        text = line.get("data") or line.get("body")
+        # Decode text if not a str but bytes or bytearray
+        if isinstance(text, (bytes, bytearray)):
+            text = text.decode("utf-8")
+
+        return [self.model(text)]
+
+
+    # def preprocess(self, data):
```
Comment (dracifer) [inline torchrecipes/paved_path/serve/handler.py:34]: we plan to enable these later?

File: torchrecipes/paved_path/serve/handler.py (hunk 1 above)
Comment (hudeven) [reply torchrecipes/paved_path/serve/handler.py:100]: Oh, right. Resolved it in diffs.

File: torchrecipes/paved_path/serve/handler.py (hunk 3 above)
Comment (hudeven) [reply torchrecipes/paved_path/serve/handler.py:34]: No, they are cleaned up in Diffs

**Review by dracifer — APPROVED** [review]

**Pull request closed without merging**